src/panels/lovelace/hui-root.ts

```diff
@@ -908,6 +908,7 @@ class HUIRoot extends LitElement {
           position: fixed;
           top: 0;
           width: var(--mdc-top-app-bar-width, 100%);
+          -webkit-backdrop-filter: var(--header-backdrop-filter, unset);
```

Comment:
Also add the non vendor prefixed key:
```suggestion
          -webkit-backdrop-filter: var(--header-backdrop-filter, unset);
          backdrop-filter: var(--header-backdrop-filter, unset);
```